src/main/java/org/cardanofoundation/conversions/exceptioni/UnsupportedConversionException.java

```diff
@@ -0,0 +1,28 @@
+package org.cardanofoundation.conversions.exceptioni;
```

Comment:
why is package called in Italian? exceptioni?

Reply:
it was a typo, it's `eccezioni` in italian, `exceptioni` it's just a typo